authorization_services/topics/service-overview.adoc

```diff
@@ -1,7 +1,13 @@
 [[_service_overview]]
 = Authorization Services
 
-{project_name} Authorization Services are based on OAuth2's User-Managed Access (UMA) Profile.
+{project_name} Authorization Services are built on top of well-known standards such as OAuth2 and User-Managed Access specifications.
 
-This section describes the different RESTful endpoints that you can interact with to enable fine-grained authorization
-for your applications and services.
+Just like OAuth2 where clients (e.g.: frontend application) cab obtain access tokens from the server using the token endpoint and use these same tokens
```

Comment:
clients (such as front end applications)

General rule: Red Hat standards require us to NOT use Latin-derived abbreviations like "e.g.". Instead, use "such as".

Same in the next line with back end services.